frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.jsx

```diff
@@ -474,18 +475,22 @@ const LoadingState = () => (
 );
 
 const NoMatchState = ({ fields }) => {
-  if (fields.length > 1) {
-    // if there is more than one field, don't name them
-    return <OptionsMessage message={t`No matching result`} />;
+  if (fields.length === 1) {
```

Comment:
Is there any reason we need to change the condition and reverse the return order?

Reply:
I don't see information about that in the ticket. Why do we change it here? Seems like we should get a confirmation from Product